src/test/java/at/ac/tuwien/kr/alpha/solver/HanoiTowerTest.java

```diff
@@ -82,12 +90,16 @@ private void testHanoiTower(int instance) throws IOException {
 	}
 
 	private void testHanoiTower(String instance) throws IOException {
+		if ("naive".equals(solverName)) {
```

Comment:
The default way to run tests conditionally seems to be `org.junit.Assume.assumeTrue(..)` and this should be placed in the `@Test` method.